docs/sources/reference/commandline/cli.md

```diff
@@ -481,8 +481,17 @@ interactively.  You can attach to the same contained process multiple times
 simultaneously, screen sharing style, or quickly view the progress of your
 daemonized process.
 
-You can detach from the container (and leave it running) with `CTRL-p CTRL-q`
-(for a quiet exit) or `CTRL-c` which will send a `SIGKILL` to the container.
+You can detach from the container and leave it running with `CTRL-p
```

Comment:
Hi, thanks for the "wonk." :-D  U missed a period in the note.  Also, no need for the future tense here.  

"Readers use technical documents to perform tasks or gather information. For readers, these activities take place in the present. Therefore, the present tense is appropriate in most cases."  -- Read Me First: A Style Guide for the Computer Industry


--->https://gist.github.com/moxiegirl/60a935fccb2e33c8e5ff

You can detach from the container and leave it running with `CTRL-p
CTRL-q` (for a quiet exit) or with `CTRL-c` if `--sig-proxy` is false.
If `--sig-proxy` is true (the default),`CTRL-c` sends a `SIGINT`
to the container.

>**Note**: A process running as PID 1 inside a container is treated
>specially by Linux: it ignores any signal without a handler installed.
>So, the process does not terminate on `SIGINT` or `SIGTERM` unless it is
>coded to do so.